x-pack/test/security_api_integration/tests/kerberos/kerberos_login.ts

```diff
@@ -92,21 +92,21 @@ export default function ({ getService }: FtrProviderContext) {
         expect(spnegoResponse.headers['www-authenticate']).to.be('Negotiate');
       });
 
-      it('AJAX requests should properly initiate SPNEGO', async () => {
+      it('AJAX requests should not initiate SPNEGO', async () => {
         const ajaxResponse = await supertest
           .get('/abc/xyz/spnego?one=two three')
           .set('kbn-xsrf', 'xxx')
           .expect(401);
 
         expect(ajaxResponse.headers['set-cookie']).to.be(undefined);
-        expect(ajaxResponse.headers['www-authenticate']).to.be('Negotiate');
+        expect(ajaxResponse.headers['www-authenticate']).to.be(undefined);
       });
     });
 
     describe('finishing SPNEGO', () => {
       it('should properly set cookie and authenticate user', async () => {
         const response = await supertest
-          .get('/internal/security/me')
+          .get('/security/account')
```

Comment:
__note:__ recently we started to treat all `/internal/....` as API requests (`canRedirectRequest === false`).